src/components/SEO/index.tsx

```diff
@@ -37,12 +45,22 @@ export const SEO: React.FC<SEOProps> = ({
   const { locale } = useLocalization()
   const lang = locale as Lang
 
-  const metaDescription = description[lang] || site.siteMetadata.description
-  const siteTitle = site.siteMetadata?.title
+  const metaDescription =
+    description[lang] || site.siteMetadata[lang].description
+  const siteTitle = site.siteMetadata[lang].title
   const socialTitle = pageTitle[lang]
     ? `${pageTitle[lang]} | ${siteTitle}`
     : siteTitle
 
+  const keywords = [
+    siteTitle,
+    "Matters",
+    "NFT",
+    "Ethereum",
+    "Blockchain",
+    "ERC-721",
+  ]
```

Comment:
do we add some keywords in `zh` as well?

Reply:
added